content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -207,7 +207,12 @@ From a shell on your instance, in the directory where you placed the `replicated
       your setup email.
 1. Secure access to the installer dashboard. We recommend at least setting up the
    simple password authentication. If you're so inclined, LDAP authentication can also be
-   configured.
+   configured using the following steps:
+      1. Navigate to the installer dashboard at https://<TFE_HOSTNAME>:8800
```

Comment:
Anything that includes angle-bracket placeholders needs to be wrapped in a code span! Otherwise it looks like an HTML tag and gets eaten by the browser. 

As a rule of thumb, almost anything that MIGHT include an angle-bracket placeholder is already a "computer word" in the first place and should probably be in a code span anyway. 

```suggestion
      1. Navigate to the installer dashboard at `https://<TFE_HOSTNAME>:8800`
```